workers/tests/integrationtests/test_api.py

```diff
@@ -29,6 +29,12 @@ def test_get_available_regions(self):
         result = geojson.loads(json.dumps(response_content))
         self.assertTrue(result.is_valid)
 
+    def test_list_indicator_layer_combinations(self):
+        url = "/indicatorLayerCombinations"
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        self.assertIsInstance(geojson.loads(response.content), dict)
```

Comment:
Since the expected response is not a GeoJSON I would not use the geojson library to parse it. Instead use the function provided by FastAPI itself.
```suggestion
        self.assertIsInstance(response.json(), dict)
```